dietpi/dietpi-software

```diff
@@ -12488,41 +12495,30 @@ _EOF_
 		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
 
 			Banner_Configuration
-
-			# Make sure user agrees to the EULA
-			G_WHIP_BUTTON_OK_TEXT='YES'
+      
+      Create_User -d /opt/papermc papermc # This creates a system user named papermc with /opt/papermc as its home dir.
+      java -jar /opt/papermc/paperclip.jar nogui
+      echo eula=true > /opt/papermc/eula.txt
+      G_WHIP_BUTTON_OK_TEXT='YES'
 			G_WHIP_BUTTON_CANCEL_TEXT='NO'
-			if G_WHIP_YESNO "Do you agree to the Minecraft EULA found at.
-\nhttps://account.mojang.com/documents/minecraft_eula"; then
-        java -jar /usr/local/bin/paper/paperclip.jar nogui
-				echo eula=true > /usr/local/bin/paper/eula.txt
-        if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate
-        \n plugins for compatibility with Bedrock Edition?"; then
-            curl https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar -o /usr/local/bin/paper/plugins/Geyser-Spigot.jar
-            curl https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar -o /usr/local/bin/paper/plugins/floodgate-bukkit.jar
-        fi
-        # Service
-			cat << '_EOF_' > /etc/systemd/system/paper.service
+      if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate
+      \n plugins for compatibility with Bedrock Edition?"; then
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar' /opt/papermc/plugins/Geyser-Spigot.jar
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar' /opt/papermc/plugins/floodgate-bukkit.jar
+      fi
+      # Service
+      cat << '_EOF_' > /etc/systemd/system/papermc.service
 [Unit]
-Description=paper (DietPi)
+Description=papermc (DietPi)
 
 [Service]
-SyslogIdentifier=papermc
-WorkingDirectory=/usr/local/bin/paper
-ExecStart=/bin/bash -c 'java -jar /usr/local/bin/paper/paperclip.jar'
+User=papermc
```

Comment:
`SyslogIdentifier=papermc` can stay. Else in system log, messages of would be prefixed with "java" currently, before last commit it would have been "bash", hence by default it is the major executable that is called via `ExecStart`. In such cases, where one does not call the software executable directly, e.g. a `papermc` binary, for sake of beauty and overview of syslog it makes sense to define `SyslogIdentifier` to prefix the syslog entries with the name of the software service 😃.